tests/unit/BusinessActions.spec.ts

```diff
@@ -2,6 +2,7 @@ import LegalServices from '@/services/legal-services'
 import { createPinia, setActivePinia } from 'pinia'
 import { useBusinessStore, useRootStore } from '@/stores'
 import { CorpTypeCd } from '@bcrs-shared-components/corp-type-module'
+import { vi } from 'vitest'
```

Comment:
I swear there's a way to do this without importing, just not sure how

Reply:
Use `vitest` instead?

Reply:
Please check latest commit. Apparently, the import was not needed whatsoever 😆

Reply:
Haha, nice!